RealmSwift/Tests/ProjectedCollectTests.swift

```diff
@@ -37,7 +31,6 @@ class ProjectedCollections: Projection<PersistedCollections> {
 }
 
 class ProjectedCollectionsTestsTemplate: TestCase {
```

Comment:
Very minor but it looks like the file name for this should be `ProjectedCollectionTests.swift`